spec/components/schemas/content/optin_request.ts

```diff
@@ -0,0 +1,62 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './base';
+import { createComponentRef } from '../../../../utils/ref';
+
+const notificationOptIn: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'optin_request',
+      },
+      medialUrl: {
+        description: 'URL for the thumbnail the user will see.',
+        type: 'string',
+        example: 'https://cdn.zenvia.com/main-picture.jpg',
+      },
+      title: {
+        description: 'The topic of the recurring notification subscription.',
+        type: 'string',
+        example: 'Weekly Sale'
+      },
+      frequency: {
+        description: 'How often the messages will be sent after user opted-in.',
```

Comment:
Eu só incluiria um `the`, e talvez usar `contact` igual nos textos do response:
`How often the messages will be sent after the contact opted-in.`

E será que esse campo não é uma boa ir também no webhook?

Reply:
@rafael-org 
FAz sentido. Inicialmente pensei que só pelo título/tópico ele já iria conseguir distinguir as diferentes _subscriptions_.
Mas vai que ele tem diferentes frequências para um mesmo assunto.

Reply:
Sim, e realmente a principio dá, mas pode ser util em alguns casos saber a frequência ainda mais se uma hora pediu semanal e outra mensal, pra saber qual que o usuário clicou.